spring-cloud-gcp-bigquery/src/test/java/org/springframework/cloud/gcp/bigquery/integration/outbound/BigQueryFileMessageHandlerIntegrationTests.java

```diff
@@ -55,6 +57,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
+@DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)
```

Comment:
Right so for this one it's because we Autowire the `BigQueryFileMessageHandler` instance, and in one test I set the dataset/table expressions to something explicit, in another test I just rely on the default.

Because I set it in the first test it modifies the state of the message handler bean so I wanted to reset it after each test.